forecasts.py

```diff
@@ -111,8 +111,6 @@ async def fetch_forecast(session: ClientSession, station: WeatherStation) -> Wea
     data = data.set_index('DATETIME')
     # Interpolate using time. (All the null values from our data insert will now be populated)
     data = data.interpolate(method='time')
-    # Filter to noon values only.
-    data = data.at_time('{}:00:00'.format(utc_noon))
```

Comment:
I'm assuming your filtering on the able in the front-end now?

Reply:
I have two normalized lookup tables in Redux. One store all model data and the other stores noon values only. And I use them depending on the situation.